layouts/partials/head.html

```diff
@@ -21,31 +21,34 @@
 <!-- Icons -->
 <link rel="apple-touch-icon-precomposed" sizes="144x144" href="{{ "img/apple-touch-icon-144-precomposed.png" | absURL }}">
 {{ if isset $.Site.Params "favicon" }}
-<link rel="shortcut icon" href="{{ $.Site.Params.favicon | absURL }}">
+  <link rel="shortcut icon" href="{{ $.Site.Params.favicon | absURL }}">
 {{ else }}
-<link rel="shortcut icon" href="{{ printf "img/favicon/%s.png" $.Site.Params.ThemeColor | absURL }}">
+  <link rel="shortcut icon" href="{{ printf "img/favicon/%s.png" $.Site.Params.ThemeColor | absURL }}">
 {{ end }}
 
 <!-- Twitter Card -->
 <meta name="twitter:card" content="summary" />
-<meta name="twitter:title" content="{{ if .IsHome }}{{ $.Site.Title }} — {{ $.Site.Params.Subtitle }}{{ else }}{{ .Title }} :: {{ $.Site.Title }} — {{ $.Site.Params.Subtitle }}{{ end }}" />
-<meta name="twitter:description" content="{{ if .IsHome }}{{ $.Site.Params.tagline }}{{ else }}{{ .Summary | plainify }}{{ end }}" />
-<meta name="twitter:site" content="{{ $.Site.BaseURL }}" />
-<meta name="twitter:creator" content="{{ .Params.Author }}" />
-<meta name="twitter:image" content="{{ with .Params.Cover }}{{ . | absURL }}{{ end }}">
+{{ if (isset $.Site.Params.Twitter "site") }}
+  <meta name="twitter:site" content="{{ $.Site.Params.Twitter.site }}" />
+{{ end }}
+<meta name="twitter:creator" content="{{ if .IsHome }}{{ $.Site.Params.Twitter.creator }}{{ else }}{{ .Params.Author }}{{ end }}" />
```

Comment:
One last thing... I'm thinking about the `twitter:creator` param. How does the `.Params.Author` match to this? When someone set `$.Site.Params.Twitter.creator`, this will show up only if you share base url. For any other page, it will show `.Params.Author` which can make no sense for the twitter card. Am I right? 🤔 

Maybe we should add something like this instead?

```suggestion
<meta name="twitter:creator" content="{{ if .IsHome }}{{ $.Site.Params.Twitter.creator }}{{ else if .Params.Twitter.author }}{{ .Params.Twitter.creator }}{{ else }}{{ .Params.Author }}{{ end }}" />
```

What do you think?

Reply:
Ah, yes sorry I missed this one! 
So my thought here was to implement a `.Params.AuthorTwitter` in /archetypes/posts.md so that an author can submit a post with their Twitter handle and that can be used. If that doesn't exist, fallback to `$.Site.Params.Twitter.creator`

```markdown
+++ 
title = "" 
date = "" 
author = "" 
authorTwitter = "" #do not include @
cover = "" 
tags = ["", ""] 
keywords = ["", ""] 
description = "" 
showFullContent = false 
+++
```

What do you think?

Reply:
> So my thought here was to implement a `.Params.AuthorTwitter` in /archetypes/posts.md so that an author can submit a post with their Twitter handle and that can be used.

👌 

> If that doesn't exist, fallback to `$.Site.Params.Twitter.creator`

Not sure, because what if you have **multiple authors**? Each author can have its own Twitter handle... Then if someone forget to set `.Param.authorTwitter` the fallback will show `.Param.Author` in the card. It will not be directly linked to the Twitter account, but it will show the real author like `John Doe`. But maybe my thinking is wrong here ;-D

Reply:
Honestly, I'm not really sure - Twitter's documentation for how it's handled isn't actually very good!

My thinking was, if I run a blog where I (primarily) post, then I would leave `.Params.AuthorTwitter` blank so it can fallback to the value set at site level. However, if I have a guest post, then I can populate that value for that author and it will show that value instead. So maybe the fallback process should be `.Params.AuthorTwitter` -> `$.Site.Params.Twitter.creator` -> `.Param.Author`

Bit of pseudo-code here:

```js
if( isset( .Params.AuthorTwitter ) ){
  .Params.AuthorTwitter
} else if( isset( $.Site.Params.Twitter.creator ) ) {
  $.Site.Params.Twitter.creator
} else {
  .Param.Author
}
```

Reply:
OK, let it be! What could possibly go wrong ;-D

Reply:
Hahaha - well it would be an easy fix at least!

I'm happy to get the `Params.AuthorTwitter` committed up. It's not too significant and at least gives people a couple of layers of fallback.

Should look _something_ like this

```twig
<meta name="twitter:creator" content="{{ if  .IsHome }}{{ $.Site.Params.Twitter.creator }}{{ else if ( isset .Params "authortwitter" ) }}{{ .Params.authorTwitter }}{{ else }}{{ .Params.Author }}{{ end }}" />
```

Reply:
Looks great!